sdk/webpubsub/azure-messaging-webpubsubservice/azure/messaging/webpubsubservice/_patch.py

```diff
@@ -292,6 +294,8 @@ def __init__(
         **kwargs  # type: Any
     ):
         # type: (...) -> None
+        if kwargs.get("port") and endpoint:
```

Comment:
What if the user already specified the port?

Reply:
yeah I had thought about that, but since I haven't documented port really, the only way that port ends here is from the connection string. So the connection string would give the port in both endpoint and in the port section, which I think is unlikely